### PR TITLE
Mv 593 confusing error message rate limits update docs

### PR DESCRIPTION
**Pull request by camillgi**
## For the Committer

- [x] I made sure that the site looks great and functions perfectly live
- [x] I ran spell check
- [x] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

File: rateLimits/messageRateLimits.md [hunk 1]
```diff
@@ -33,13 +33,14 @@ All Bandwidth messaging products are rate limited in some fashion. The default r
...
 
 ### 403 – LIMIT
-| Code                | Message                                                                                                     |
-|:--------------------|:------------------------------------------------------------------------------------------------------------|
-| message-rate-limit  | You can send ${0} messages per ${1} seconds calculated as the average over ${2} seconds. Your rate is: ${3} |
-| multi-message-limit | The number of messages [${0}] in the multi message request is greater than the limit [${1}].                |
+| Code                        | Message                                                                                                        |
+|:----------------------------|:---------------------------------------------------------------------------------------------------------------|
+| message-rate-limit-overall  | You can send ${0} messages per ${1} seconds calculated as the average over ${2} seconds. Your rate is: ${3}    |
```
Comment (jdhudson3) [inline rateLimits/messageRateLimits.md]: Did these error codes change? Looks like they are both return the error code "message-rate-limit".

Comment (camillgi) [reply rateLimits/messageRateLimits.md]: I changed them here https://github.com/Bandwidth/Development/pull/1683 but I haven't merged it yet

Comment (camillgi) [reply rateLimits/messageRateLimits.md]: Just realized I change the message but not the error code category in that PR. Need to fix that

Comment (camillgi) [reply rateLimits/messageRateLimits.md]: Just realized I change the message but not the error code category in that PR. Need to fix that

Comment (camillgi) [reply rateLimits/messageRateLimits.md]: Okay this is updated in the other PR now

**Pull request merged**